envelop/__init__.py

```diff
@@ -17,9 +17,10 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from __future__ import unicode_literals
+from six.moves.urllib.parse import urlparse
+from builtins import str as text
```

Comment:
Makes sense. Another alternative could be to include `future>=0.16.0` in `requirements.txt`? `builtins` is installed through that.